### PR TITLE
Containerized testing of DTS

### PR DESCRIPTION
**Pull request by grafnu**

### CODE REVIEW

File: bin/setup_stack [hunk 1]
```diff
@@ -135,6 +142,19 @@ docker kill forch-faucet-corp || true
...
+    if [[ $dts == localhost ]]; then
+        docker rm -f forch-dts || true
+        echo Running local DTS container...
+        DTS_CID=$(docker run -d --privileged --name forch-dts -v $ROOT/inst/forch-dts:/root/inst forch/dts)
+        DTS_IP=$(docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' $DTS_CID)
+        vxlan="vxlan $DTS_IP"
+    else
+        vxlan="vxlan $dts"
+    fi
+    echo Targetting DTS container at $vxlan
```
Comment (henry54809) [inline bin/setup_stack:155]: Typo

**Comment by grafnu** [comment]
Sorry -- I'm not seeing it... what's the typo exactly?

File: bin/setup_stack (hunk 1 above)
Comment (henry54809) [reply bin/setup_stack:155]: Targeting? I googled and targetting doesn't seem to be valid.

**Comment by grafnu** [comment]
Huh. TIL.
https://www.future-perfect.co.uk/grammar-tips/just-plain-wrong/is-it-targetted-or-targeted
-- I just assumed that because "get --> getting" it would targetting, but
something something english is complicated.

**Review by henry54809 — APPROVED** [review]

**Review by anurag6 — APPROVED** [review]

**Pull request merged**